### PR TITLE
Implement sstable_info command

### LINKED ISSUE #76: Implement "getSSTableInfo"
We need scylla-jmx support for REST call sstable_info, wrapping and transforming data into JMX consumable format that can be used by nodetool. 

See /scylladb/scylla-tools-java#105
See /scylladb/scylla#4726

### PR DESCRIPTION
**Pull request by elcallio**
Fixes #76 

Implements JMX level call for "sstable_info" REST api command.

Requires seastar patch:
 json: Make date formatter use RFC8601/RFC3339 format

Requires scylla patch set "Implement `sstable_info` API command (info on sstables)"

Forwards call to REST sstable_info and packs the data
into CompositeData for JMX consumption.

### CODE REVIEW

File: src/main/java/org/apache/cassandra/service/PerTableSSTableInfo.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import java.util.List;
+
+import javax.management.openmbean.CompositeData;
+import javax.management.openmbean.OpenDataException;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.sun.jmx.mbeanserver.MXBeanMapping;
+
+@SuppressWarnings("restriction")
+@XmlRootElement
```
Comment (penberg) [inline src/main/java/org/apache/cassandra/service/PerTableSSTableInfo.java]: What is this `XmlRootElement` annotation about?

Comment (elcallio) [reply src/main/java/org/apache/cassandra/service/PerTableSSTableInfo.java]: To enable automatic marshalling of object <-> json via java REST client.

File: src/main/java/org/apache/cassandra/service/StorageServiceMBean.java [hunk 2]
```diff
@@ -872,4 +873,8 @@ default public void reloadLocalSchema() {
      * Sets the hinted handoff throttle in kb per second, per delivery thread.
      */
     public boolean resumeBootstrap();
+    
+    public List<CompositeData> getSSTableInfo(String keyspace, String table);
+
+    public List<CompositeData> getSSTableInfo();
```
Comment (penberg) [inline src/main/java/org/apache/cassandra/service/StorageServiceMBean.java]: I don't see anything on `nodetool` side that would call into these new methods. Will there be another pull request against that repository to enable this functionality or am I missing something obvious?

Comment (elcallio) [reply src/main/java/org/apache/cassandra/service/StorageServiceMBean.java]: Yes. https://github.com/scylladb/scylla-tools-java/pull/106

File: src/main/java/org/apache/cassandra/service/PerTableSSTableInfo.java (hunk 1 above)
Comment (penberg) [reply src/main/java/org/apache/cassandra/service/PerTableSSTableInfo.java]: Obviously... I think I have forgotten the power of Java enterprise APIs!

Thanks for the clarification!

Comment (elcallio) [reply src/main/java/org/apache/cassandra/service/PerTableSSTableInfo.java]: Powerful apis they are... Powerful apis.

File: src/main/java/org/apache/cassandra/service/StorageServiceMBean.java (hunk 2 above)
Comment (penberg) [reply src/main/java/org/apache/cassandra/service/StorageServiceMBean.java]: Aah, thanks!
